app/src/main/java/com/framgia/soundclound/data/source/remote/LoadAsync.java

```diff
@@ -0,0 +1,75 @@
+package com.framgia.soundclound.data.source.remote;
+
+import android.os.AsyncTask;
+
+import com.framgia.soundclound.data.source.TrackDataSource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.logging.Logger;
```

Comment:
ERROR: Unused import: java.util.logging.Logger. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports